backend/src/fun_forecast_backend/shared/dataclasses.py

```diff
@@ -1,9 +1,13 @@
 import datetime
 from dataclasses import dataclass
 from enum import Enum
-from typing import Optional
+from typing import Optional, Any
 
 
+@dataclass
+class ResponseObject:
```

Comment:
The frontend framework expects the ApiResponse to have a shape of:
```
{
data?: Any
meta?: Any
errors?: Any
}
```
But I wasn't sure how to allow optional keys like that.

Reply:
Good question - I think we can make them nullable and you can check if the value is present

Reply:
How does one make a key nullable? I did a cursory google and didn't see a simple answer

Reply:
We should make an ABC response object and inherit it to specific responses imo. I can work on that